### PR TITLE
introduce openshift template

### PR DESCRIPTION
**Pull request by riuvshin**
introduce openshift template

### CODE REVIEW

**Comment by riuvshin** [comment]
[test]

**Comment by riuvshin** [comment]
@steveeJ this can be changed, I've moved it to dist

File: dist/openshift/cincinnati.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                  key: registry
+                  name: cincinnati
+            - name: REPOSITORY
+              valueFrom:
+                configMapKeyRef:
+                  key: repository
+                  name: cincinnati
+          args: ["-vvv", "--address", "$(ADDRESS)", "--registry", "$(REGISTRY)", "--repository", "$(REPOSITORY)"]
+          imagePullPolicy: Always
+          name: cincinnati
```
Comment (steveej) [inline dist/openshift/cincinnati.yaml]: I think we should differentiate between the graph-builder and the policy-engine in the name. The two should already be able to work together so we can add the policy-engine to the template too and expose either the latter or both.

**Review by steveej — CHANGES_REQUESTED** [review]
I'm not confident about the syntax of the template but I recommend to add the policy-engine as a second container and chain the two like in the examples of #20

**Comment by steveej** [comment]
Please rebase this onto current master so it doesn't mess up the CI permissions.

File: dist/openshift/cincinnati.yaml (hunk 1 above)
Comment (riuvshin) [reply dist/openshift/cincinnati.yaml]: done

**Comment by riuvshin** [comment]
[test]

File: dist/openshift/cincinnati.yaml [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      deploymentconfig: cincinnati
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: cincinnati-policy-engine
+  spec:
+    ports:
+      - protocol: TCP
+        port: 80
+        targetPort: 8080
```
[the remark this answers is not in the document]
Comment (riuvshin) [reply dist/openshift/cincinnati.yaml]: yeah, moved it to parameters

**Review by steveej — APPROVED** [review]

**Pull request merged**